public/assets/icons/slider__arrow-right.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="50" height="16" viewBox="0 0 50 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M1 7C0.447715 7 4.82823e-08 7.44772 0 8C-4.82823e-08 8.55228 0.447715 9 1 9L1 7ZM49.7071 8.70711C50.0976 8.31659 50.0976 7.68342 49.7071 7.2929L43.3431 0.928936C42.9526 0.538411 42.3195 0.538411 41.9289 0.928936C41.5384 1.31946 41.5384 1.95263 41.9289 2.34315L47.5858 8L41.9289 13.6569C41.5384 14.0474 41.5384 14.6805 41.9289 15.0711C42.3195 15.4616 42.9526 15.4616 43.3431 15.0711L49.7071 8.70711ZM1 9L49 9L49 7L1 7L1 9Z" fill="#FB5B11"/>
```

Comment:
we'd better use fill="currentColor" here, because we have arrows with different colours